Makefile.am

```diff
@@ -3885,6 +3885,12 @@ endif
 # Integration tests #
 #####################
 
+if DISTRO_FEDORA
```

Comment:
I think these changes are not needed and it should be enough to include --with-subid in https://github.com/SSSD/sssd/blob/master/contrib/ci/configure.sh

Reply:
Hm... If CI doesn't use `intgcheck-prepare` target, then why this change had any effect on CI...

Reply:
Ah, it is used, but changing value of `(INTGCHECK_CONFIGURE_FLAGS)` should be easier, indeed. Thanks.

Reply:
Fixed.